### PR TITLE
feat: integrate ks configurations

### PR DESCRIPTION
**Pull request by namgold**

### CODE REVIEW

File: src/hooks/useKyberSwapConfig.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      setKyberswapConfigs(null)
+      const fetches = storeChainIds.map(async (chainId) => {
+        try {
+          const { data, error } = await getKyberswapConfiguration({ chainId })
+          if (data)
+            return {
+              chainId,
+              result: data,
+            }
+          if (error)
```
Comment (nguyenhoaidanh) [inline src/hooks/useKyberSwapConfig.ts:31]: remove if error

File: src/state/application/hooks.ts [hunk 2]
```diff
@@ -60,8 +61,20 @@ export function useClients(): {
...
+    dataClient: ApolloClient<NormalizedCacheObject>
+    blockClient: ApolloClient<NormalizedCacheObject>
+  }[]
+  try {
+    result = activeNetwork.map((network) => ({
+      dataClient: kyberswapConfig[network.chainId].client,
+      blockClient: kyberswapConfig[network.chainId].blockClient,
+    }))
+  } catch {
+    debugger
```
Comment (nguyenhoaidanh) [inline src/state/application/hooks.ts:76]: rm

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (namgold) [reply src/hooks/useKyberSwapConfig.ts:31]: I want to leave it, for any change in logic later would be easily.

Comment (nguyenhoaidanh) [reply src/hooks/useKyberSwapConfig.ts:31]: image
same result. no need if error

Comment (namgold) [reply src/hooks/useKyberSwapConfig.ts:31]: I know what u r mentioning to. I prefer to leave it as is.

Comment (nguyenhoaidanh) [reply src/hooks/useKyberSwapConfig.ts:31]: :dungz

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**